pyproject.toml

```diff
@@ -1,11 +1,11 @@
 [tool.poetry]
 name = "cdip_connector"
-version = "1.2.7"
+version = "1.2.7.2"
```

Comment:
I'll update this accordingly once the PR is approved
I'm using this version in GFW at the moment.